app/views/admin/dfc_product_imports/index.html.haml

```diff
@@ -3,4 +3,26 @@
 
 = render partial: 'spree/admin/shared/product_sub_menu'
 
-watch this space
+%p= t('.catalog_url', catalog_url: @catalog_url)
+%p= t('.enterprise', enterprise_name: @enterprise.name)
+%br
+
+= form_with url: main_app.import_admin_dfc_product_imports_path do |form|
+  -# This is a very inefficient way of holding a json blob. Maybe base64 encode or store as a temporary file
```

Comment:
I think that base64 would just make it bigger. Temporary file is an option. ActiveStorage is good at that. But let's keep this for now until we hit performance problems. The endpoints will probably have pagination at some point, too. And maybe we will want the user to trigger the "load more" action on this page. The we combine it here or store several blobs.